lib/restrictions.js

```diff
@@ -73,6 +73,25 @@ export async function insert({restriction} = {}) {
   return record;
 }
 
+/**
+ * Inserts multiple restrictions into database.
+ *
+ * @param {object} options - The options to use.
+ * @param {Array} options.restrictions - An array of restrictions.
+ * @returns {Promise} - Settles once the operation completes.
+ */
+export async function bulkInsert({restrictions} = []) {
+  const operations = [];
+  for(const restriction of restrictions) {
+    const now = Date.now();
+    operations.push({
+      insertOne: {document: {restriction, meta: {created: now, updated: now}}}
+    });
+  }
+  const collection = database.collections['resource-restriction-restriction'];
+  await collection.bulkWrite(operations);
```

Comment:
Can we use `insertMany` instead? We can use the same date and meta for all of them as well.

```suggestion
  const now = Date.now();
  const meta = {created: now, updated: now};
  const records = restrictions.map(restriction => ({restriction, meta}));
  // allow unordered writes
  const writeOptions = {...database.writeOptions, ordered: false};
  const collection = database.collections['resource-restriction-restriction'];
  await collection.insertMany(records, writeOptions);
```

Not sure what we want to do with duplicates here, but for now I guess we're throwing. We may want to change that in the future.